pallets/sto/src/lib.rs

```diff
@@ -235,13 +282,15 @@ decl_module! {
                 venue_id,
                 start,
                 end,
-                frozen: false,
+                status: FundraiserStatus::Live,
+                minimum_investment
             };
 
             let id = FundraiserCount::mutate(offering_asset, |id| mem::replace(id, *id + 1));
             <Fundraisers<T>>::insert(offering_asset, id, fundraiser.clone());
+            FundraiserNames::insert(offering_asset, id, fundraiser_name.clone());
```

Comment:
I don't see the use of the fundraiser name, As per the current implementation, it can be dedup.

Reply:
Not sure what you mean?

Fundraiser names, just like other names, are only for offchain usage. I see no reason to enforce any kind of uniqueness on them.

Reply:
Can't we just emit in an event? Instead of storing as well.